PowerShellTools/ServiceManagement/PowershellHostProcessException.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PowerShellTools.ServiceManagement
+{
+    [Serializable]
+    public class PowershellHostProcessException : Exception

```

Comment:
If you want to invent an exception type dedicated for this purpose(to let caller catch so that caller knows something wrong during spawning the process), at least you need reconsider the custom exception name to something like "PowerShellHostProcessInitializeException", so that it is not giving people impression that is generic exception from remote process. Although in such scenario, I'd more prefer to just do regular error handling check, and show VS dialog if needed to notify user, instead of instantiate an exception giving to VS to handle.